### PR TITLE
zero-code/python: correct the OTEL_PYTHON_DISABLED_INSTRUMENTATIONS description

### PR DESCRIPTION
**Pull request by xrmx**
The name you have to pass to it is not the package name but the entry point name you have under the project.entry-points.opentelemetry_instrumentor table in the package pyproject.toml.

Also add grpc to the example as it's a notable example of this difference.

### CODE REVIEW

**Review by svrnm — APPROVED** [review]

**Review by emdneto — APPROVED** [review]

File: content/en/docs/zero-code/python/configuration.md [hunk 1]
```diff
@@ -165,13 +165,16 @@ in too much or unwanted data.
...
+`pyproject.toml` file.
 
 For example, if your Python program uses the `redis`,`kafka-python` and `grpc`
 packages, by default the agent will use the
 `opentelemetry-instrumentation-redis`,
-`opentelemetry-instrumentation-kafka-python` and `opentelemetry-instrumentation-grpc` packages to instrument them. To
-disable this, you can set
+`opentelemetry-instrumentation-kafka-python` and
+`opentelemetry-instrumentation-grpc` packages to instrument them. To disable
+this, you can set
```
Comment (emdneto) [inline content/en/docs/zero-code/python/configuration.md:179]: The entrypoint for kafka-python is just `kafka`no?

Comment (xrmx) [reply content/en/docs/zero-code/python/configuration.md:179]: Ah!

File: content/en/docs/zero-code/python/configuration.md [hunk 2]
```diff
@@ -165,12 +165,16 @@ in too much or unwanted data.
...
 packages, by default the agent will use the
-`opentelemetry-instrumentation-redis` and
-`opentelemetry-instrumentation-kafka-python` packages to instrument them. To
-disable this, you can set
-`OTEL_PYTHON_DISABLED_INSTRUMENTATIONS=redis,kafka-python`.
+`opentelemetry-instrumentation-redis`,
+`opentelemetry-instrumentation-kafka-python` and
+`opentelemetry-instrumentation-grpc` packages to instrument them. To disable
+this, you can set
+`OTEL_PYTHON_DISABLED_INSTRUMENTATIONS=redis,kafka-python,grpc_client`.
```
Comment (xrmx) [inline content/en/docs/zero-code/python/configuration.md:180]:
```suggestion
`OTEL_PYTHON_DISABLED_INSTRUMENTATIONS=redis,kafka,grpc_client`.
```

File: content/en/docs/zero-code/python/configuration.md (hunk 1 above)
Comment (xrmx) [reply content/en/docs/zero-code/python/configuration.md:179]: Fixed thanks

**Pull request merged**